backend/organization/models/status.py

```diff
@@ -70,3 +70,94 @@ class Meta:
 
     def __str__(self):
         return "Project status  %s" % (self.name)
+
+
+def project_type_image_path(instance, filename):
+    return "project_type_icons/{}/{}".format(instance.id, filename)
+
+
+class ProjectTypes(models.Model):
+    name = models.CharField(
+        help_text="Name of the project type",
+        verbose_name="Name",
+        max_length=512,
+        null=True,
+        blank=True,
+    )
+
+    name_de_translation = models.CharField(
+        help_text="Translation of name column",
+        verbose_name="Name DE translation",
+        max_length=512,
+        null=True,
+        blank=True,
+    )
+
+    help_text = models.TextField(
+        help_text="Small helper text that is shown under the name in the shrae project interface",
+        verbose_name="Helptext",
+        max_length=512,
+        null=True,
+        blank=True,
+    )
+
+    help_text_de = models.TextField(
+        help_text="Translation of help_text column",
+        verbose_name="Helptext DE translation",
+        max_length=512,
+        null=True,
+        blank=True,
+    )
+
+    icon = models.FileField(
```

Comment:
Is icon going to be an image or something else? Can we use imagefield here instead?

Reply:
I used file field to allow for svg icons